docker-compose.yml

```diff
@@ -13,16 +13,13 @@ services:
 
   firefly-app:
     image: firefly-iii
-    build:
-      context: .
+    build: .
     environment:
-      - INIT_DATABASE=yes
```

Comment:
Does this command also work when the production environment is set? I am not entirely sure, but I thought I saw that not happening.

Although, even if it does: I think that for the safety of production data, it might be better to run a manual command once to get it al started. If you forget to remove this environment variable that would otherwise mean that you loose all your data, which, in my opinion, is a bigger problem.